libs/community/langchain_community/retrievers/zep_cloud.py

```diff
@@ -46,17 +52,23 @@ class ZepCloudRetriever(BaseRetriever):
 
     api_key: str
     """Your Zep API key."""
-    zep_client: Zep
+    zep_client: Any
     """Zep client used for making API requests."""
-    zep_client_async: AsyncZep
+    zep_client_async: Any
     """Async Zep client used for making API requests."""
-    session_id: str
-    """Zep session ID."""
+    session_id: Optional[str]
+    """Session ID to search. Deprecated."""
+    session_ids: Optional[List[str]]
+    """List of session IDs to search."""
+    user_id: Optional[str]
+    """User ID to search."""
     top_k: Optional[int]
     """Number of items to return."""
-    search_scope: SearchScope = "messages"
+    min_score: Optional[float]
+    """Minimum score to return."""
+    search_scope: str = "facts"
     """Which documents to search. Messages or Summaries?"""
```

Comment:
does this docstring need to be updated?

also do we need to change the default? or could we just document the new option

Reply:
@baskaryan Thanks, addressed both. We will keep the default value of "messages" to not break any existing use cases.